bears/r/RLintBear.py

```diff
@@ -19,7 +19,7 @@ class RLintBear:
     """
     Checks the code with ``lintr``.
     """
-    LANGUAGES = {'R'}
+    LANGUAGES = {'R >=3.1.1'}
```

Comment:
Where do I do it then? I tried changing the Requirements dict in the source code of RLintBear in the last commit and I was told that that wasn't the right place either. I saw the generated bear docs repo and compared it with the source code and the place where I made the change now is the place that has all the metadata shown in the generated bear docs repo